integration_test/Bindings/Python/dialects/om.py

```diff
@@ -140,10 +153,15 @@
   print(e)
 
 for (name, field) in obj:
+  # location from om.class.field @child, %0 : !om.class.type<@Child>
   # CHECK: name: child, field: <circt.dialects.om.Object object
-  # CHECK: name: field, field: 42
-  # CHECK: name: reference, field: ('Root', 'x')
-  print(f"name: {name}, field: {field}")
+  # CHECK-SAME: loc: loc("-":32:7)
```

Comment:
nit: it is a little confusing this one uses CHECK-SAME but the below lines just use CHECK and include the `loc:` check. Any reason not to do it the same for all of them?

Reply:
This one was required because of the `Object` address that is printed on previous line, tried to avoid adding a regex there. 
I agree it makes sense to follow  this same pattern for the following checks too.

Reply:
oh i see, that makes sense.